1/contrib/openshift/configuration/jobs/OpenShift Sample/config.xml

```diff
@@ -57,11 +57,13 @@
       <namespace>test</namespace>
       <testTag>latest</testTag>
       <prodTag>prod</prodTag>
-      <testStream>origin-nodejs-sample</testStream>
-      <prodStream>origin-nodejs-sample</prodStream>
+      <testStream>origin-ruby-sample</testStream>
+      <prodStream>origin-ruby-sample</prodStream>
       <authToken></authToken>
     </com.openshift.jenkins.plugins.pipeline.OpenShiftImageTagger>
 
+    <!--
+    FIXME: There is no such deployment config created by sample-app.
```

Comment:
there isn't?  this job is supposed to be paired w/ this template, right?
https://github.com/openshift/origin/blob/master/examples/jenkins/application-template.json#L63-L67